USAGE_DATA.adoc

```diff
@@ -27,3 +27,5 @@ Disable::
 `odo preference set ConsentTelemetry false`
 
 Note: If earlier the `ConsentTelemetry` preference was enabled, then the data will be collected about the disabling of the preference.
+
+Alternatively you can disable telemetry by setting `ODO_DISABLE_TELEMETRY` environment variable to `false`.
```

Comment:
```suggestion
Alternatively you can disable telemetry by setting `ODO_DISABLE_TELEMETRY` environment variable to `true`.
```
Simply based on what you've mentioned in PR description:
> introduces ODO_DISABLE_TELEMETRY env variable. When this is set to true odo will skip the telemetry question and it won't send any telemetry data out even if odo preference set consentTelemetry true.

I haven't looked at the code yet.

Reply:
oh, good catch.